recipes/lodepng/all/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.0)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_library(lodepng "source_subfolder/lodepng.h" "source_subfolder/lodepng.cpp")
+
+include(GNUInstallDirs)
+install(TARGETS lodepng)
```

Comment:
```suggestion
install(TARGETS lodepng
        RUNTIME DESTINATION bin
        LIBRARY DESTINATION lib
        ARCHIVE DESTINATION lib)
```

explicit

Reply:
GNUInstallDirs already defines standard GNU installation directories. (see https://cmake.org/cmake/help/latest/command/install.html#installing-targets)

Reply:
I know, that's why I wrote "explicit".

Reply:
You include `GNUInstallDirs`, but don't use the variables.
The suggestion should use the `CMAKE_INSTALL_BINDIR` and `CMAKE_INSTALL_LIBDIR` variables.